Nupkg.props

```diff
@@ -14,8 +14,8 @@
     <!-- <PackageVersion>Defined in GlobalStaticVersion.props</PackageVersion> -->
     <Authors>Microsoft</Authors>
     <Owners>Microsoft,AppInsightsSdk</Owners>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?LinkID=510709</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance Condition="$(OS) == 'Windows_NT'">true</PackageRequireLicenseAcceptance>
```

Comment:
@Dmitry-Matveev can you test this condition on line 3 instead?
I was thinking about this, and our Linux builds don't need to create nuget packages.